salesforce-backed-customer-insight-agent/sk_connectors.py

```
@@ -0,0 +1,184 @@
+        # Fallback defaults for this project
+        defaults = {"slack": "slack", "salesforce": "salesforce"}
+        guess = defaults.get(service.lower())
+        if guess:
+            print(f"ℹ️  Defaulting connector '{guess}' for {service}")
+        return guess
+
+    def get_authorization_url(self, service: str, user_identifier: str) -> str:
+        try:
+            connector_name = self._guess_connector_name(service, user_identifier)
+            if not connector_name:
+                raise RuntimeError(f"No connector available for service '{service}'")
```

Ruff (0.14.1)

Abstract `raise` to an inner function

(TRY301)